mParticle-Apple-SDK/MPEnums.h

```diff
@@ -198,8 +198,10 @@ typedef NS_ENUM(NSUInteger, MPKitInstance) {
     /** Kit code for Adobe */
     MPKitInstanceAdobe = 124,
     /** Kit code for Instabot */
-    MPKitInstanceInstabot = 123
-    
+    MPKitInstanceInstabot = 123,
+    /** Kit code for Carnival */
+    MPKitInstanceCarnival = 99
```

Comment:
Would you mind moving this up to be in the correct place, sorted by the kit code?